reorg security service naming (#715)
aerogear/mobile-docs: modules/ROOT/pages/_partials/app-security/proc-calling-the-app-security-clientInit.adoc

> = Coding your mobile app to use the {app-security-service} service

Reviewer: ```suggestion
= Implementing the {app-security-service} service in your mobile app
```
